### DISCUSSION TITLE
Calls to action in the list/collapsed view

### DISCUSSION THREAD

**Discussion by danicaso-vizzuality** [Vizzuality/mangrove-atlas#290] [General]
Hi all, 

In our last call, you talked about starting with the cards collapsed. That way, we can see the list of datasets/layers at first glance. We found that approach great, so we added some elements playing around this way of navigating: 

https://www.figma.com/proto/HksskREDktKK9OBHUfaxQ1/Mangrove-Watch-v02.2?page-id=0%3A1&node-id=31%3A2370&viewport=-14%2C662%2C0.20956838130950928&scaling=min-zoom

We brought the bottom of the cards (_Download data_ and _More info_) to the top. That way, they are accessible from the list view. The download buttons will deploy a tooltip to choose between download local data or the entire dataset. 
A little detail: The layer colors are shown on the chart, so hitting the _Show layer_ button would also expand the given card, as it will do hitting anywhere on the upper area of the card.

Looking forward to hearing your thoughts.
Thank you!

**Comment by dhakelila** [comment]
13th July meeting notes about this task: 

- Liked the idea proposed. 
- Improvement: When the user clicks on one of the options, instead of directly download the file, a modal window will be open to show some information regarding the file you are about to download. Dani will do the changes to the design.

**Comment by lyubovpencheva** [comment]
This is an old one.
